mne/beamformer/tests/test_lcmv.py

```diff
@@ -139,12 +140,30 @@ def test_lcmv():
         max_stc = stc_max_power.data[idx]
         tmax = stc.times[np.argmax(max_stc)]
 
-        assert_true(0.09 < tmax < 0.11, tmax)
+        assert_true(0.04 < tmax < 0.11, tmax)
```

Comment:
tmax is 0.085 sec for the surface forward model and 0.105 sec for the volume forward model. I will put the test to 0.08.